File: Fundamentos de Programación/Ejercicio 2.12.cpp
#include <iostream>

int main()
{
    int h = 0;
    float t = 0, sal = 0;

    std::cout<<"Ingrese el número de horas: ";
    std::cin >>h;
    std::cout<<"\nIngrese la tarifa por horas: ";
    std::cin >>t;
    sal = h*t;
    std::cout<<"El salario es de "<<sal;
}

File: Fundamentos de Programación/Ejercicio 2.12.py
while True:
    try:
        h = int(input('Ingrese el número de horas: '))
        t = float(input('Ingrese la tarfa por hora: '))
        break
    except:
        continue

sal = h*t
print('El salario es de ' + str(sal))

exit()
